lib/skin.js

```diff
@@ -290,31 +290,35 @@ module.exports = function (gulp, gutil) {
       if (!parent) {
         var parentId = this.getParentId();
         if (parentId) {
-          let asyncFunction = (parentId1) => {
-              return new Promise((resolve, reject) => {
-                  var server = getServer();
-                  checkTheme = require('./check-theme-enabled.js')(gulp, gutil);
-                  checkTheme.checkThemeEnabled(server.serverUrl(), server.pluginToken(), {
-                    themeId: parentId1,
-                    debugMode: gutil.env.debug
-                  }, function(err, isThemeEnabled) {
-                        if (err) {
-                          process.exitCode = 1;
-                          putils.logWarning(gutil, "ERROR");
-                          throw err;
-                        }
-                        resolve(isThemeEnabled);
-                    });
-              })
-            }
-            
-            let syncFunc = sp(asyncFunction)
-            const value = syncFunc(parentId)
-            if(value === 'true') {
+          if (parentId.includes("theme")) {
+            let asyncFunction = (parentId1) => {
```

Comment:
why is asyncFunction a _let_ instead of a _const_?  Is it reassigned somwhere?

Reply:
I will change it to const